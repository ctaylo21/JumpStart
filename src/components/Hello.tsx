import React from 'react';

export interface HelloProps {
  compiler: string;
  framework: string;
}

const Hello = ({ compiler, framework }: HelloProps): React.ReactElement => (
  <h2>
    Hello from {compiler} and {framework}!
  </h2>
);

export default Hello;
