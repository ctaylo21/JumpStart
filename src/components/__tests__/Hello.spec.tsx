import * as React from 'react';
import * as renderer from 'react-test-renderer';
import Hello from '../Hello';

it('SubjectToBeTested renders correctly', () => {
  const props = {
    compiler: 'newCompiler',
    framework: 'react',
  };
  const tree = renderer.create(<Hello {...props} />).toJSON();
  expect(tree).toMatchSnapshot();
});
