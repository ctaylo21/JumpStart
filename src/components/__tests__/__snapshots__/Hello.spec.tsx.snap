// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Hello renders correctly 1`] = `
<h1>
  Hello from 
  newCompiler
   and 
  react
  !
</h1>
`;
